### DISCUSSION TITLE
Please support mongodb

### DISCUSSION THREAD

**Discussion by Charuru** [ubicloud/ubicloud#1033] [Ideas]
Please support mongodb, looking good on the Postgres offering

**Comment by ozgune** [comment]
Hi @Charuru - thank you for your input!

We had a clarification question to better understand your requirements. What would be your motivation for using MongoDB on Ubicloud? Is it the ability to run on bare metal, on Hetzner, price/performance? Any info you could provide on your requirements would greatly help us.

On our side, we're evaluating ways to provide managed MongoDB in the future. MongoDB's Server Side Public License (SSPL) prohibits us from offering it as a managed service unless we have a licensing agreement with MongoDB, the company. There are options such as FerretDB (https://github.com/FerretDB/FerretDB), but understandably, they have a few differences re: feature coverage.

**Reply by Charuru** [reply]
My main motivation is price/perf and the ability to run on Hetzner (at least semi-managed). Most of my servers are on Hetzner but I'm currently using DigitalOcean for managed MongoDB, but I would love to bring it to Hetzner for better pricing and latency. 

Oh [FerretDB](https://github.com/FerretDB/FerretDB) is certainly interesting though I've never seen it before. I'm not sure how will it'll work for me.

Thanks a lot!

**Reply by byucesoy** [reply]
Hi @Charuru,

You might find this interesting. We recently collaborated with FerretDB on [this blog post](https://blog.ferretdb.io/add-mongodb-compatibility-ubicloud-managed-postgres/). It explains how you can add MongoDB compatibility on top of Ubicloud Postgres using FerretDB.
